rocky/rocky/views/ooi_list.py

```diff
@@ -77,16 +76,18 @@ def post(self, request: HttpRequest, *args, **kwargs) -> HttpResponse:
             return self._delete_oois(selected_oois, request, *args, **kwargs)
 
         if action == PageActions.UPDATE_SCAN_PROFILE.value:
-            return self._set_scan_profiles(selected_oois, request, *args, **kwargs)
+            scan_profile = request.POST.get("scan-profile").lower()
```

Comment:
This could fail since `request.POST.get("scan-profile")` returns `Optional[str]`. Perhaps first validate wether this input is given